modules/monitoring-accessing-third-party-monitoring-uis.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * monitoring/accessing-third-party-monitoring-uis-and-apis.adoc
+
+:_content-type: CONCEPT
+[id="accessing-third-party-monitoring-uis"]
+= Accessing third-party monitoring UIs
+
+[role="_abstract"]
+{product-title} does not provide or support direct access to third-party web user interfaces (UIs) for components in the monitoring stack such as Grafana, Alertmanager, or Thanos Querier. As an alternative to third-party UIs, navigate to the *Observe* section of the {product-title} web console to access Metrics, Alerting, and Dashboard UIs for these platform components.
```

Comment:
since 'Thanos Ruler' is in  [accessing-third-party-monitoring-uis-and-apis.adoc](https://github.com/openshift/openshift-docs/pull/41384/files#diff-2d645834c95c663c4aecf714e32dc362a918117b35f8866e55508e2e80a34709), it should be added here also.  Grafana UI is still accessible till OCP 4.10. 
does not provide or support direct access to third-party web user interfaces (UIs) for components in the monitoring stack such as Alertmanager, Thanos Ruler, or Thanos Querier.

Reply:
Added Thanos ruler